network-area-diagram-viewer/src/main/resources/svg.js

```diff
@@ -0,0 +1,448 @@
+
+// TODO(Luma) consider three winding transformers
```

Comment:
The first thing I think of is not moving them when moving a neighbouring node. But then we need to rotate the transformer so that it's properly oriented, hence moving all 3 edges end also.

Reply:
agree, maybe we could deal with this in a separate PR?